charts/harvester-cloud-provider/values.yaml

```diff
@@ -69,3 +69,21 @@ global:
   cattle:
     systemDefaultRegistry: ""
     clusterName: ""
+
+kube-vip:
+  enabled: true
+  image:
+    repository: ghcr.io/kube-vip/kube-vip-iptables
```

Comment:
can we please help to check the required image for air-gap env and private registry config. https://github.com/rancher/rke2/blob/master/scripts/build-images#L86

Reply:
Done